plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergTableProperties.java

```diff
@@ -64,6 +71,11 @@ public IcebergTableProperties(IcebergConfig icebergConfig)
                         "File system location URI for the table",
                         null,
                         false))
+                .add(integerProperty(
+                        FORMAT_VERSION_PROPERTY,
+                        "Iceberg table format version for newly created tables",
+                        icebergConfig.getFormatVersion(),
+                        false))
```

Comment:
add validation that it's between min and max supported

(`Consumer<Integer> validation`)